### DISCUSSION TITLE
Improving whitespace in unminified output — would PR be worth submitting?

### DISCUSSION THREAD

**Discussion by danielfdickinson** [gohugo-ananke/ananke#445] [3 - Ideas]
Hi, and thank you for this great theme.

I have a personal preference for my generated HTML to be reasonably 'tight' even before minification (that is, where it's not more hassle than it's worth, avoiding having extra newlines or trailing whitespace). This makes the development HTML more readable (IMO).

However, I do not want to carry such updates in my fork as I like what you do, and plan on updating regularly. I'm therefore asking whether a PR that improves the whitespace in the emitted HTML when not minifying would be accepted, or if I should avoid spending the time on that.

I have other contributions I plan on making, but this one is least likely to be accepted as a change, I think, so I thought I'd ask before doing.

**Comment by danielfdickinson** [comment]
@regisphilibert @budparr Any opinion on this? The lack of response suggests it's a no go, but before declaring it dead I'd like to be sure you as maintainers have seen this discussion, and it's not just a case of not realizing this discussion item was here.

**Reply by regisphilibert** [reply]
Hi Daniel. Sorry for the late response. We'll get back to you today. This does need a bit of thinking.

**Comment by danielfdickinson** [comment]
No problem, just wanted to make sure you'd seen it. I'll wait for your response. If it helps, my plan is do it chunks, not one massive PR (unless you'd prefer that).
